packages/editor-ui/src/components/MainHeader/CollaborationPane.vue

```diff
@@ -1,22 +1,28 @@
 <script setup lang="ts">
-import { COLLABORATION_HEARTBEAT_INTERVAL } from '@/constants';
 import { useUsersStore } from '@/stores/users.store';
 import { useWorkflowsStore } from '@/stores/workflows.store';
 import { useCollaborationStore } from '@/stores/collaboration.store';
 import { onBeforeUnmount } from 'vue';
 import { onMounted } from 'vue';
 import { computed, ref } from 'vue';
+import { TIME } from '@/constants';
 
 const collaborationStore = useCollaborationStore();
 const usersStore = useUsersStore();
 const workflowsStore = useWorkflowsStore();
 
-const heartbeatInterval = COLLABORATION_HEARTBEAT_INTERVAL;
-const heartbeatTimer = ref(null as null | NodeJS.Timeout);
+const HEARTBEAT_INTERVAL = 5 * TIME.MINUTE;
+const heartbeatTimer = ref(null as null | number);
 
-const activeUsers = computed(() => {
+const activeUsersSorted = computed(() => {
+	const currentWorkflowUsers = (collaborationStore.getUsersForCurrentWorkflow ?? []).map(
+		(userInfo) => userInfo.user,
+	);
+	const owner = currentWorkflowUsers.find((user) => user.globalRoleId === 1);
 	return {
-		default: (collaborationStore.getUsersForCurrentWorkflow || []).map((userInfo) => userInfo.user),
+		defaultGroup: owner
+			? [owner, ...currentWorkflowUsers.filter((user) => user.id !== owner?.id)]
```

Comment:
The optional operator should be unncessary, as we checked that owner is indeed defined

```suggestion
			? [owner, ...currentWorkflowUsers.filter((user) => user.id !== owner.id)]
```